backend/jest.config.ts

```diff
@@ -0,0 +1,9 @@
+export default {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  collectCoverageFrom: ['src/*.{js,ts}', '!**/node_modules/**'],
+  modulePaths: ['<rootDir>/src'],
+  testMatch: ['<rootDir>/tests/**/*.test.ts'],
+  setupFiles: ['<rootDir>/test-resources/env-vars.js'],
```

Comment:
TODO: Add remaining required environment variables for testing (e.g. `ENCRYPTION_KEY`).

Reply:
hey or maybe we add those variables which the test actually needs, I mean define the variables when the corresponding tests need that.

Reply:
@Aashish-Upadhyay-101 Sounds good!